patches/chrome-installer-mac-signing-pipeline.py.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/installer/mac/signing/pipeline.py b/chrome/installer/mac/signing/pipeline.py
+index ac3cd9f6b28570410730f8aaaf98b3f7ce468320..6d49ec610df9b9550e1eb2d6febd431eb6e2a724 100644
+--- a/chrome/installer/mac/signing/pipeline.py
++++ b/chrome/installer/mac/signing/pipeline.py
+@@ -85,7 +85,6 @@ def _package_installer_tools(paths, config):
+         'dirdiffer.sh',
+         'dirpatcher.sh',
+         'dmgdiffer.sh',
+-        'keystone_install.sh',
```

Comment:
The scripts includes `keystone_install.sh` as an additional tool here, but in `chrome/installer/mac/BUILD.gn` the file is only copied when chrome-branded:
`if (is_chrome_branded) {
    sources += [ "keystone_install.sh" ]
  }`